@utahdts/utah-design-system/package.json

```diff
@@ -68,13 +65,14 @@
   },
   "homepage": "https://github.com/utahdts/utah-design-system",
   "dependencies": {
+    "@floating-ui/react-dom": "^2.1.2",
```

Comment:
dependencies should be locked and not have the caret in front of the version. ie "^19.1.0" should be "19.1.0". Otherwise it will use any version after 19 instead of exactly 19.1.0.